packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -137,7 +137,7 @@ export default function FeedFilters(props: FeedFiltersProps): ReactElement {
           </Modal.Body>
           <Modal.Body
             view={FilterMenuTitle.ContentTypes}
-            className="relative overflow-hidden"
+            className="relative overflow-x-hidden overscroll-none"
```

Comment:
enable vertical scroll and disable overscroll on iOS so it does not show the overflowed "coming soon"